src/gui/sdlmain.cpp

```diff
@@ -3211,6 +3208,11 @@ static void set_output(Section *sec, bool should_stretch_pixels)
 
 	sdl.render_driver = section->Get_string("texture_renderer");
 	lowcase(sdl.render_driver);
+	if (sdl.render_driver != "auto") {
+		if (SDL_SetHint(SDL_HINT_RENDER_DRIVER,
+		                sdl.render_driver.c_str()) == SDL_FALSE)
+			LOG_WARNING("SDL: Failed to set texture renderer driver; falling back to automatic selection");
+	}
```

Comment:
Suggestion a set of braces { ... } for the inner `if` body (I visually couldn't really tell which level we were at).

Reply:
Let's include the config value in the report:

`Failed to set the '%s' texture renderer driver; ...`

Maybe the value is coming from a higher-up conf or their primary, and this will help them figure it out (and will also help us when people copy-and-paste one-line error messages into issue tickets)

Reply:
Done and done.